src/Model/ResourceField.php

```diff
@@ -59,12 +59,17 @@ public function getCMSFields()
             $fields->dataFieldByName('ReadableName')
                 ->setAttribute('placeholder', $this->Name);
 
-            $summary = $fields->dataFieldByName('ShowInSummaryView');
-            $detail = $fields->dataFieldByName('ShowInDetailView');
-            $duplicates = $fields->dataFieldByName('RemoveDuplicates');
+            $summary = $fields->dataFieldByName('ShowInSummaryView')
+                ->addExtraClass('VisibilityOptions__Option');
```

Comment:
I mentioned this in another PR but convention is that CSS classes are all lower case. If we're not doing this already I think we should be from now on. In this case it should be `visibility-options__option`.

Reply:
Thanks, I'll update.

Reply:
Done :)